enhancements/dns/coredns-egressfirewall-integration.md

```diff
@@ -513,7 +484,7 @@ allow rules are supported by Cilium.
 ### gRPC connection between OVN-K and CoreDNS
 
 Communication between OVN-K master and CoreDNS happens over a gRPC connection rather than the proposed `EgressFirewallDNSName` CR. Whenever there's a DNS
-lookup for a DNS Name which is used in an EgressFirewall rule and the IPs associated with DNS name changes, then CoreDNS sends this information to the
+lookup for a DNS Name which is used in an EgressFirewall rule and the IP addresses associated with DNS name changes, then CoreDNS sends this information to the
 OVN-K master. After the underlying ACL rules are updated the OVN-K master responds to the same CoreDNS pod with an OK message. Then the CoreDNS
 pod responds to the original DNS lookup request.
 
```

Comment:
I couldn't add a comment in the Pros/Cons section of gRPC, but I like the idea of gRPC for this enhancement.  It allows the proper function for egress firewall without interfering with other DNS lookups.

Reply:
The gRPC based solution will also need a new plugin which will inspect every DNS lookup. Without that it will not be possible to know whether a lookup is requested for a particular DNS. The difference here would be how CoreDNS communicates the information with OVN-K, which is via a gRPC connection. However, with new OVN interconnect architecture where there can be multiple OVN-K masters, this solution will not scale well.